source/common/http/http1/codec_impl.cc

```diff
@@ -783,13 +842,16 @@ void ServerConnectionImpl::handlePath(RequestHeaderMap& headers, unsigned int me
   // Absolute URLS in CONNECT requests will be rejected below by the URL class validation.
   if (!codec_settings_.allow_absolute_url_ && !is_connect) {
     headers.addViaMove(std::move(path), std::move(active_request.request_url_));
-    return;
+    return true;
   }
 
   Utility::Url absolute_url;
   if (!absolute_url.initialize(active_request.request_url_.getStringView(), is_connect)) {
     sendProtocolError(Http1ResponseCodeDetails::get().InvalidUrl);
-    throw CodecProtocolException("http/1.1 protocol error: invalid url in request line");
+    ASSERT(codec_status_.ok());
+    ENVOY_LOG_MISC(info, "SETTING PROTOCOL ERROR");
```

Comment:
did you plan to leave this in? If so, can you include the URL?

Reply:
done -- but actually this was useful for debugging. What do you think about adding logging statements when the `codec_status_` sets an error? 

This ends up getting logged when `onData` calls dispatch, but that was too late for any codec level test debugging

Reply:
When debugging something in the heat of the moment, more logs are great, as long as the quantity of logs doesn't become a problem. I only suggest that if there's a URL to log, we include that.